apps/prairielearn/src/pages/sideNavSettings/sideNavSettings.ts

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+
+const router = Router();
+
+router.put('/', async (req, res) => {
+  const new_state = Boolean(req.body.side_nav_expanded);
+  req.session.side_nav_expanded = new_state;
+  res.status(200).json({
+    side_nav_expanded: new_state,
+  });
```

Comment:
Nit, the client won't ever look at the response so we can just send a [204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/204).

```suggestion
  req.session.side_nav_expanded = Boolean(req.body.side_nav_expanded);
  res.sendStatus(204);
```